Update readme no support for Pyquil version 3.0 (#79)
PennyLaneAI/pennylane-forest: README.rst

as well as the following Python packages:

* `PennyLane <http://pennylane.readthedocs.io/>`__
* `pyQuil <https://pyquil-docs.rigetti.com/en/stable/>`__ >=2.16, <2.28.3

> We do not support the latest PyQuil version 3.0 at the moment, but we are working on it.

Reviewer: ```suggestion
Note that the latest PyQuil version 3.0 is not currently supported.
```

Reviewer: Just in case!
